docs/scripts/generate_api_docs.py

```diff
@@ -76,6 +76,19 @@
     ],
 }
 
+def is_documented_method(obj):
```

Comment:
`is_documented_method` sounds like we are checking if the method has a docstring, probably go with `should_document_method`?